### ISSUE TITLE
loss decline but accuracy near to 0

### ISSUE THREAD

**Issue by bjlgcxc** [summerlvsong/Aggregation-Cross-Entropy#25]
I train a model(CRNN) base on dataset synth90k, through the loss decline step by step, the accuracy is near to 0 all the time. What casue this problem?

**Comment by bjlgcxc** [comment]
@summerlvsong

**Comment by dqfx1949** [comment]
I also find this problem in the training of crnn,have you solved it?

**Comment by Hubert2102** [comment]
+1

**Comment by bjlgcxc** [comment]
> I also find this problem in the training of crnn,have you solved it?

not yet

**Comment by Zhang-O** [comment]
> I train a model(CRNN) base on dataset synth90k, through the loss decline step by step, the accuracy is near to 0 all the time. What casue this problem?

make sure your decode method works correctly. you'd better test it by some your customed fake data.
